website/content/en/preview/AWS/user-data.md

```diff
@@ -46,6 +47,37 @@ spec:
     karpenter.sh/discovery: my-cluster
 ```
 
+This example adds SSH keys to allow remote login to the node (replace *my-authorized_keys* with your key file):
```

Comment:
can you add the blurb about SSM and ec2-instance-connect here too? 

```
You can use Session Manager (SSM) or EC2 Instance Connect to gain shell access to Karpenter nodes.
See [Node NotReady](https://karpenter.sh/preview/troubleshooting/#node-notready) troubleshooting for an example of starting an SSM session from the command line or [EC2 Instance Connect](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-connect-set-up.html) documentation to connect to nodes using SSH.
```

Reply:
Done.